requirements.txt

```diff
@@ -1,11 +1,11 @@
 edx-rest-api-client==1.6.0
+edx-rest-api-client==1.6.0
```

Comment:
@schenedx because it was so disorganized before that no one noticed. I'll remove the duplicate line. Going forward, scripts should specify their own requirements. There's no need to install all of these requirements to run one of the scripts in here.